subschedule.go

```diff
@@ -115,12 +122,28 @@ type SubscriptionScheduleCurrentPhase struct {
 	StartDate int64 `json:"start_date"`
 }
 
+// SubscriptionScheduleBillingThresholds is a structure representing the
+// parameters allowed to control billing thresholds for a subscription item.
+type SubscriptionScheduleBillingThresholds struct {
+	AmountGTE               int64 `json:"amount_gte"`
+	ResetBillingCycleAnchor bool  `json:"reset_billing_cycle_anchor"`
+}
+
```

Comment:
This should not be here. Sharing was done before and used the subscription's one `SubscriptionBillingThresholdsParams` and sharing is what the spec says to do (and what java does). Si you should remove that class and use `SubscriptionBillingThresholdsParams` instead.